### PR TITLE
Feture moblie portfolio project

### PR DESCRIPTION
**Pull request by sushen**
1. HTML Sclaton
2. add CSS
3. add GitHub workflow

### CODE REVIEW

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @sushen , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request :heavy_multiplication_x: 
Invalid


You have submitted a project, that is not documented professionally due to the absence of a README file. 
- [x] Kindly add a  README to your project and request another review.
`Hint: You may consider using the template provided by Microverse` [here](https://github.com/microverseinc/readme-template) `and personalize it to your project.`

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Sushen Biswas poritipholio
```
Comment (algerina) [inline README.md]: - [ Optional] Your project will look more professional if you fix the spelling errors, especially in the project name `Sushen Biswas Portfolio`

**Review by algerina — APPROVED** [review]
Hi @sushen , 
Good job so far! 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:


## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous TSE


 
 ## Optional suggestions
 
 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @Algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (sushen) [reply README.md]: I fixed it, Thanks

**Pull request merged**